x-pack/legacy/plugins/infra/public/containers/source/source.tsx

```diff
@@ -108,11 +121,13 @@ export const useSource = ({ sourceId }: { sourceId: string }) => {
     [apolloClient, sourceId]
   );
 
-  const derivedIndexPattern = useMemo(
-    () => ({
-      fields: source ? source.status.indexFields : [],
-      title: source ? `${source.configuration.logAlias}` : 'unknown-index',
-    }),
+  const createDerivedIndexPattern = useCallback(
```

Comment:
Could you elaborate a bit why `useCallback()` is used here?

Reply:
Actually I don't think it needs to be there, I got overzealous